src/modules/trpc/trpc-client.ts

```diff
@@ -43,7 +44,7 @@ export const trpcClient = createTRPCNext<AppRouter>({
           },
         }),
         httpBatchLink({
-          url: "/api/trpc",
+          url: `${env.NEXT_PUBLIC_BASE_PATH}/api/trpc`,
```

Comment:
**question**: Won't this break if `env.NEXT_PUBLIC_BASE_PATH` is not provided? It can be `undefined` AFAIK

Reply:
Yes, is true. The app should not start if those variables are not set.